.travis.yml

```diff
@@ -3,12 +3,12 @@ sudo: required
 
 install:
   - docker run --rm --privileged linuxkit/binfmt:v0.8
-  - docker run --name buildkit --rm -d --privileged -p 1234:1234 $REPO_SLUG_ORIGIN --debug --addr tcp://0.0.0.0:1234 --oci-worker-gc=false
+  - docker run --name buildkit -d --privileged -p 1234:1234 $REPO_SLUG_ORIGIN --debug --addr tcp://0.0.0.0:1234 --oci-worker-gc=false
   - sudo docker cp buildkit:/usr/bin/buildctl /usr/bin/
   - export BUILDKIT_HOST=tcp://0.0.0.0:1234
 
 after_failure:
-  - docker logs buildkit
+  - docker ps -a && docker logs buildkit
```

Comment:
I'm not sure `docker ps -a` check is meaningful

Reply:
Just to show what state the container is in. Eg. if it crashed.